planner/core/plan.go

```diff
@@ -390,6 +390,11 @@ func (p *baseLogicalPlan) buildLogicalPlanTrace() *tracing.LogicalPlanTrace {
 	return planTrace
 }
 
+func buildPhysicalOptimizeTraceInfo(p Plan, prop string) *tracing.PhysicalOptimizeTraceInfo {
```

Comment:
```suggestion
func buildPhysicalOptimizeTraceInfo(p PhysicalPlan, prop string) *tracing.PhysicalOptimizeTraceInfo {
```

Reply:
It should be `LogicalPlan`, has been modified.